packages/backend/src/videos/videos.controller.ts

```diff
@@ -19,15 +18,9 @@ export class VideosController{
         return this.videosService.getAll(param);
     }
 
-    @Get('category')
-    getAllCategoryList(){
-        return this.videosService.getAllCategoryList();
-    }
-
-    @ApiParam({ name: 'id', type:'string' })
-    @Get('/:params')
-    getStream(@Query() id: string ,@Res() res, @Req() req) {
-        this.videosService.getStream(id, res, req);
+    @Get("category")
+    getAllCategoryList(@Query() param:ResponseVersionCategoryDTO) {

```

Comment:
Skąd Response w nazwie? To te ma nic wspólnego z odpowiedzią

Reply:
![image](https://user-images.githubusercontent.com/6808853/95385531-6d10fa80-08ee-11eb-943d-b02491e72521.png)

Reply:
sugerowałem się tym to jest wersja odpowiedzi która będzie zwracana jeśli nazwa jest zła daj znać